orchestrator/src/itn_orchestrator/main.go

```diff
@@ -269,15 +269,15 @@ func main() {
 			continue
 		}
 		cmd := *commandOrComment.command
-		if prevAction != nil && prevAction.Name() != cmd.Action {
```

Comment:
I don't think it's necessary to move this code below, let's revert this small change

Reply:
https://github.com/o1-labs/mina-perf-testing/pull/2/commits/15c96ba2fa3b501ca3cd6a6be74c1cf10f8d68c8